plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -76,13 +76,30 @@ func isVolumeConflict(volume api.Volume, pod *api.Pod) bool {
 			}
 		}
 	}
+	if volume.RBD != nil {
+		mon := volume.RBD.CephMonitors
+		pool := volume.RBD.RBDPool
+		image := volume.RBD.RBDImage
+
+		manifest := &(pod.Spec)
```

Comment:
this var name can change to `podspec`.  Manifest is leftover from the old beta API.

The others can be improved, too, though it can also be a separate PR.